src/components/common/modals/common-modal.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react'
+import { Modal } from 'react-bootstrap'
+import { Trans } from 'react-i18next'
+import { ForkAwesomeIcon, IconName } from '../fork-awesome/fork-awesome-icon'
+import { ShowIf } from '../show-if/show-if'
+
+export interface CommonModalProps {
+  show: boolean
+  onHide: () => void
+  title: string
+  closeButton?: boolean
+  icon?: IconName
+}
+
+export const CommonModal: React.FC<CommonModalProps> = ({ show, onHide, title, closeButton, icon, children }) => {
```

Comment:
You're using the Trans component but not useTranslation

Reply:
You obviously found a bug in the current ErrorModal code. Well, then it'll be fixed after merging this PR. :)